fem/nonlininteg.cpp

```diff
@@ -247,114 +448,1655 @@ void NeoHookeanModel::AssembleH(const DenseMatrix &J, const DenseMatrix &DS,
             }
 }
 
+double TMOPHyperelasticModel001::EvalW(const DenseMatrix &Jpt) const
```

Comment:
My preference will be to change the name as TMOP models doesn't have to be `elastic`.